diff --git a/dist/index-316c5b13.js b/dist/index-316c5b13.js
index a551e3a43dd53c98c83ac2a4855f703c4bf95dea..71a8cacd116179c2e7c64f95270e05b860bacb50 100644
--- a/dist/index-316c5b13.js
+++ b/dist/index-316c5b13.js
@@ -5967,7 +5967,8 @@ function create_if_block$3(ctx) {
 		c() {
 			div = element("div");
 			if_block.c();
-			attr(div, "class", "icon-container svelte-1xw6u9q");
+      attr(div, "class", "icon-container svelte-1xw6u9q");
+      attr(div, "part", "sidebar-heading-img");
 		},
 		m(target, anchor) {
 			insert(target, div, anchor);
@@ -6485,7 +6486,8 @@ function create_if_block_6$1(ctx) {
 			t2 = space();
 			div2 = element("div");
 			t3 = text(t3_value);
-			attr(div0, "class", "icon-container svelte-1qwmck3");
+      attr(div0, "class", "icon-container svelte-1qwmck3");
+      attr(div0, "part", "mobile-icon-container");
 			attr(div1, "class", "header-heading svelte-1qwmck3");
 			attr(div2, "class", "mobile-subheader svelte-1qwmck3");
 			attr(div3, "class", "flex flex-column justify-center w-full svelte-1qwmck3");
@@ -6595,7 +6597,8 @@ function create_else_block_1(ctx) {
 		c() {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*icon*/ ctx[14])) attr(img, "src", img_src_value);
-			attr(img, "alt", "logo");
+      attr(img, "alt", "logo");
+      attr(img, "part", "mobile-icon-img");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
