diff --git a/dist/index-f9d2990f.js b/dist/index-f9d2990f.js
index cf2bab77d336f20305dcf9d2d803922d0928fb58..121d705fbfa7aafdc63781cc5a993147be112d26 100644
--- a/dist/index-f9d2990f.js
+++ b/dist/index-f9d2990f.js
@@ -33963,6 +33963,7 @@ function create_if_block_5$1(ctx) {
 			div2 = element("div");
 			t3 = text(t3_value);
 			attr(div0, "class", "icon-container svelte-7qgvqp");
+      attr(div0, "part", "mobile-icon-container");
 			attr(div1, "class", "header-heading svelte-7qgvqp");
 			attr(div2, "class", "mobile-subheader svelte-7qgvqp");
 			attr(div3, "class", "flex flex-column justify-center w-full svelte-7qgvqp");
@@ -34073,6 +34074,7 @@ function create_else_block_1(ctx) {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*icon*/ ctx[14])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
+      attr(img, "part", "mobile-icon-img");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
