diff --git a/dist/Index-a3f5e101.js b/dist/Index-a3f5e101.js
index b83bc176bbb566b43260f929889bf34032d73574..955635fe81ffbf9e2982db178d9405f844325c85 100644
--- a/dist/Index-a3f5e101.js
+++ b/dist/Index-a3f5e101.js
@@ -1191,6 +1191,7 @@ function create_default_slot(ctx) {
 			button1 = element("button");
 			t7 = text(t7_value);
 			attr(div0, "class", "icon-container flex justify-center items-center svelte-10lq1b5");
+      attr(div0, "part", "sidebar-heading-img");
 			attr(h4, "class", "svelte-10lq1b5");
 			attr(p, "class", "svelte-10lq1b5");
 			attr(button0, "class", "button-neutral-solid-b rounded svelte-10lq1b5");
diff --git a/dist/index-4d2b1bb4.js b/dist/index-4d2b1bb4.js
index 53027db853683c877007fa913cd44b19d959f9fe..358a2d295fae597d1cea659642f9d563cfe34702 100644
--- a/dist/index-4d2b1bb4.js
+++ b/dist/index-4d2b1bb4.js
@@ -5576,6 +5576,7 @@ function create_else_block$1(ctx) {
 			if (!src_url_equal(img.src, img_src_value = /*logo*/ ctx[3] || /*icon*/ ctx[2])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
 			attr(img, "class", "svelte-kwt38y");
+      attr(img, "part", "sidebar-heading-image");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
@@ -5704,6 +5705,7 @@ function create_fragment$5(ctx) {
 			t9 = space();
 			div5 = element("div");
 			attr(div0, "class", "icon-container flex svelte-kwt38y");
+      attr(div0, "part", "sidebar-heading-img")
 			attr(h4, "class", "subheading svelte-kwt38y");
 			attr(p, "class", "description svelte-kwt38y");
 			attr(div1, "class", "indicator relative svelte-kwt38y");
