diff --git a/dist/index-316c5b13.js b/dist/index-316c5b13.js
index a551e3a43dd53c98c83ac2a4855f703c4bf95dea..70e5da22af5df2329c9f058c98b819f4efb8c139 100644
--- a/dist/index-316c5b13.js
+++ b/dist/index-316c5b13.js
@@ -5968,6 +5968,7 @@ function create_if_block$3(ctx) {
 			div = element("div");
 			if_block.c();
 			attr(div, "class", "icon-container svelte-1xw6u9q");
+      attr(div, "part", "sidebar-heading-img");
 		},
 		m(target, anchor) {
 			insert(target, div, anchor);
@@ -6486,6 +6487,7 @@ function create_if_block_6$1(ctx) {
 			div2 = element("div");
 			t3 = text(t3_value);
 			attr(div0, "class", "icon-container svelte-1qwmck3");
+      attr(div0, "class", "mobile-icon-container");
 			attr(div1, "class", "header-heading svelte-1qwmck3");
 			attr(div2, "class", "mobile-subheader svelte-1qwmck3");
 			attr(div3, "class", "flex flex-column justify-center w-full svelte-1qwmck3");
@@ -6596,6 +6598,7 @@ function create_else_block_1(ctx) {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*icon*/ ctx[14])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
+      attr(img, "alt", "mobile-icon-img");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
