diff --git a/dist/index-cfcb95b8.js b/dist/index-cfcb95b8.js
index 8a91ad360aa97a8801a4028c494f811f6eb62c2e..259db2acbe775ee9aefdb23177eb4b818c2d4120 100644
--- a/dist/index-cfcb95b8.js
+++ b/dist/index-cfcb95b8.js
@@ -3734,7 +3734,7 @@ function create_else_block_1$1(ctx) {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*iconLoaded*/ ctx[11])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
-			attr(img, "class", "svelte-1xw3izs");
+      attr(img, "class", "svelte-1xw3izs");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
@@ -5577,7 +5577,8 @@ function create_else_block$1(ctx) {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*logo*/ ctx[3] || /*icon*/ ctx[2])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
-			attr(img, "class", "svelte-kwt38y");
+      attr(img, "class", "svelte-kwt38y");
+      attr(img, "part", "sidebar-heading-img");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
@@ -5705,7 +5706,8 @@ function create_fragment$5(ctx) {
 			div4 = element("div");
 			t9 = space();
 			div5 = element("div");
-			attr(div0, "class", "icon-container flex svelte-kwt38y");
+      attr(div0, "class", "icon-container flex svelte-kwt38y");
+      attr(div0, "part", "sidebar-heading-img");
 			attr(h4, "class", "subheading svelte-kwt38y");
 			attr(p, "class", "description svelte-kwt38y");
 			attr(div1, "class", "indicator relative svelte-kwt38y");
@@ -7104,7 +7106,8 @@ function create_default_slot(ctx) {
 			t5 = space();
 			button = element("button");
 			t6 = text(t6_value);
-			attr(div0, "class", "icon-container flex justify-center items-center svelte-1sfp8g8");
+      attr(div0, "class", "icon-container flex justify-center items-center svelte-1sfp8g8");
+      attr(div0, "part", "sidebar-heading-img");
 			attr(h4, "class", "svelte-1sfp8g8");
 			attr(p, "class", "svelte-1sfp8g8");
 			attr(button, "class", "button-neutral-solid rounded svelte-1sfp8g8");
