diff --git a/dist/index-40d7e725.js b/dist/index-40d7e725.js
index b7561ff57cefb33f44cd4be378d7a370bce611b6..21a038111592639847e986c84e3cf70f2f146dcc 100644
--- a/dist/index-40d7e725.js
+++ b/dist/index-40d7e725.js
@@ -5939,7 +5939,8 @@ function create_if_block$3(ctx) {
 		c() {
 			div = element("div");
 			if_block.c();
-			attr(div, "class", "icon-container svelte-1xw6u9q");
+      attr(div, "class", "icon-container svelte-1xw6u9q");
+      attr(div, "part", "sidebar-heading-img");
 		},
 		m(target, anchor) {
 			insert(target, div, anchor);
@@ -6457,7 +6458,8 @@ function create_if_block_6$1(ctx) {
 			t2 = space();
 			div2 = element("div");
 			t3 = text(t3_value);
-			attr(div0, "class", "icon-container svelte-1pz9kpx");
+      attr(div0, "class", "icon-container svelte-1pz9kpx");
+      attr(div0, "part", "mobile-icon-container");
 			attr(div1, "class", "header-heading svelte-1pz9kpx");
 			attr(div2, "class", "mobile-subheader svelte-1pz9kpx");
 			attr(div3, "class", "flex flex-column justify-center w-full svelte-1pz9kpx");
@@ -6567,7 +6569,8 @@ function create_else_block_1(ctx) {
 		c() {
 			img = element("img");
 			if (!src_url_equal(img.src, img_src_value = /*icon*/ ctx[14])) attr(img, "src", img_src_value);
-			attr(img, "alt", "logo");
+      attr(img, "alt", "logo");
+      attr(img, "part", "mobile-icon-img");
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
