diff --git a/dist/index.js b/dist/index.js
index b5cbcb44b3bdd60b2b91cb705450f0dfdc758615..b5acfbaf6dced81232d6a8a0654ee5408a5560d5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -11,12 +11,13 @@ export default function vitePluginSvgr({ svgrOptions, esbuildOptions, include =
             if (filter(id)) {
                 const { transform } = await import("@svgr/core");
                 const { default: jsx } = await import("@svgr/plugin-jsx");
+                const { default: svgo } = await import("@svgr/plugin-svgo");
                 const filePath = id.replace(postfixRE, "");
                 const svgCode = await fs.promises.readFile(filePath, "utf8");
                 const componentCode = await transform(svgCode, svgrOptions, {
                     filePath,
                     caller: {
-                        defaultPlugins: [jsx],
+                        defaultPlugins: [svgo.default, jsx],
                     },
                 });
                 const res = await transformWithEsbuild(componentCode, id, {
