diff --git a/dist/es5/subproviders/etherscan.js b/dist/es5/subproviders/etherscan.js
index 4866d2eeba8949ab0b0063310bd2d471abd5826a..73b92ea2f5f91a3a1f256f121d713f1063f7cb36 100644
--- a/dist/es5/subproviders/etherscan.js
+++ b/dist/es5/subproviders/etherscan.js
@@ -23,7 +23,7 @@
  * - eth_getBalance
  * - eth_listTransactions (non-standard)
  */
-var xhr = process.browser ? require('xhr') : require('request');
+var xhr = require('xhr');
 
 var inherits = require('util').inherits;
 
diff --git a/dist/es5/subproviders/rpc.js b/dist/es5/subproviders/rpc.js
index 1b7157514703803b7a6d7cdf4fc9642274ec0571..5c1146de8c742d1bf644a245a3808d06fdb085ce 100644
--- a/dist/es5/subproviders/rpc.js
+++ b/dist/es5/subproviders/rpc.js
@@ -1,6 +1,6 @@
 "use strict";
 
-var xhr = process.browser ? require('xhr') : require('request');
+var xhr = require('xhr');
 
 var inherits = require('util').inherits;
 
diff --git a/subproviders/etherscan.js b/subproviders/etherscan.js
index c653788755b87f11b8e59d4ead20851679538afc..c3df1aca161aab8979eaef3d48580c767ae1d493 100644
--- a/subproviders/etherscan.js
+++ b/subproviders/etherscan.js
@@ -22,7 +22,7 @@
  * - eth_listTransactions (non-standard)
  */
 
-const xhr = process.browser ? require('xhr') : require('request')
+var xhr = require('xhr');
 const inherits = require('util').inherits
 const Subprovider = require('./subprovider.js')
 
diff --git a/subproviders/rpc.js b/subproviders/rpc.js
index fca5e02257515b18bf8780b33481d61e962c4af5..02ccf8b558a0de84cdd43bafe8e5cea14c0b4559 100644
--- a/subproviders/rpc.js
+++ b/subproviders/rpc.js
@@ -1,4 +1,4 @@
-const xhr = process.browser ? require('xhr') : require('request')
+var xhr = require('xhr');
 const inherits = require('util').inherits
 const createPayload = require('../util/create-payload.js')
 const Subprovider = require('./subprovider.js')
