diff --git a/node_modules/@react-native-community/toolbar-android/js/ToolbarAndroid.web.js b/node_modules/@react-native-community/toolbar-android/js/ToolbarAndroid.web.js
index 31f0017..d006a30 100644
--- a/node_modules/@react-native-community/toolbar-android/js/ToolbarAndroid.web.js
+++ b/node_modules/@react-native-community/toolbar-android/js/ToolbarAndroid.web.js
@@ -9,15 +9,4 @@
 
 'use strict';
 
-let m;
-try {
-  // This is an optional dependency. However, webpack might still show warning message.
-  m = require('react-native/Libraries/Components/UnimplementedViews/UnimplementedView');
-} catch {
-  // If failed to find react-native UnimplementedView, just returns an empty module so
-  // this won't cause bundling related error, however any subsequent attempt of using this module
-  // in such situation shall caused compile-error because it really should not be used for web.
-  m = {};
-}
-
-module.exports = m;
+module.exports = function () {};
