diff --git a/node_modules/@toruslabs/customauth-react-native-sdk/RNCustomAuthSdk.podspec b/node_modules/@toruslabs/customauth-react-native-sdk/RNCustomAuthSdk.podspec
index 9329966..51f0a91 100644
--- a/node_modules/@toruslabs/customauth-react-native-sdk/RNCustomAuthSdk.podspec
+++ b/node_modules/@toruslabs/customauth-react-native-sdk/RNCustomAuthSdk.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.license      = "MIT"
   # s.license    = { :type => "MIT", :file => "FILE_LICENSE" }
   s.author       = { "Torus Labs" => "hello@tor.us" }
-  s.platform     = :ios, "10.0"
+  s.platform     = :ios, "11.0"
   s.source       = { :git => "https://github.com/torusresearch/customauth-react-native-sdk.git", :branch => "master" }

   s.source_files = "ios/*.{h,m,swift}"
