diff --git a/node_modules/@web3-onboard/core/dist/index.js b/node_modules/@web3-onboard/core/dist/index.js
index 53dcf03..a6e1914 100644
--- a/node_modules/@web3-onboard/core/dist/index.js
+++ b/node_modules/@web3-onboard/core/dist/index.js
@@ -5572,7 +5572,7 @@ function instance$a($$self, $$props, $$invalidate) {
 	let connectingErrorMessage;
 	let windowWidth;
 	let scrollContainer;
-	const walletToAutoSelect = autoSelect && walletModules.find(({ label }) => label.toLowerCase() === autoSelect.label.toLowerCase());
+	let walletToAutoSelect = autoSelect && walletModules.find(({ label }) => label.toLowerCase() === autoSelect.label.toLowerCase());
 
 	// ==== SELECT WALLET ==== //
 	async function selectWallet({ label, icon, getInterface }) {
@@ -5691,6 +5691,11 @@ function instance$a($$self, $$props, $$invalidate) {
 
 			// user rejected account access
 			if (code === ProviderRpcErrorCode.ACCOUNT_ACCESS_REJECTED) {
+        if (autoSelect){
+          deselectWallet();
+          walletToAutoSelect = null;
+          setStep('selectingWallet');
+        }
 				$$invalidate(2, connectionRejected = true);
 				return;
 			}
