diff --git a/node_modules/@web3-onboard/core/dist/index.js b/node_modules/@web3-onboard/core/dist/index.js
index 713647c..1a4d26e 100644
--- a/node_modules/@web3-onboard/core/dist/index.js
+++ b/node_modules/@web3-onboard/core/dist/index.js
@@ -4639,6 +4639,7 @@ function create_else_block$2(ctx) {
 			if (!src_url_equal(img.src, img_src_value = /*logo*/ ctx[3] || /*icon*/ ctx[2])) attr(img, "src", img_src_value);
 			attr(img, "alt", "logo");
 			attr(img, "class", "svelte-1o88d5s");
+      attr(img, "part", "sidebar-heading-img")
 		},
 		m(target, anchor) {
 			insert(target, img, anchor);
