diff --git a/node_modules/core-util-is/lib/util.js b/node_modules/core-util-is/lib/util.js
index ff4c851..6e5a20d 100644
--- a/node_modules/core-util-is/lib/util.js
+++ b/node_modules/core-util-is/lib/util.js
@@ -100,7 +100,7 @@ function isPrimitive(arg) {
 }
 exports.isPrimitive = isPrimitive;
 
-exports.isBuffer = Buffer.isBuffer;
+exports.isBuffer = require('buffer').Buffer.isBuffer;
 
 function objectToString(o) {
   return Object.prototype.toString.call(o);
