diff --git a/node_modules/jest-environment-jsdom/build/index.js b/node_modules/jest-environment-jsdom/build/index.js
index 78d8feb..e99aaf9 100644
--- a/node_modules/jest-environment-jsdom/build/index.js
+++ b/node_modules/jest-environment-jsdom/build/index.js
@@ -86,6 +86,8 @@ class JSDOMEnvironment {
 
     global.ArrayBuffer = ArrayBuffer; // Node's error-message stack size is limited at 10, but it's pretty useful
     // to see more than that when a test fails.
+    global.Uint8Array = Uint8Array; // this patch is fixing Uint8Array redefined by JSDOM.
+    // TextileCrypto is failing without this override
 
     this.global.Error.stackTraceLimit = 100;
     (0, _jestUtil().installCommonGlobals)(global, config.globals); // Report uncaught errors.
