diff --git a/node_modules/lottie-react-native/src/ios/LottieReactNative/ContainerView.swift b/node_modules/lottie-react-native/src/ios/LottieReactNative/ContainerView.swift
index cd8ed10..2ab65b4 100644
--- a/node_modules/lottie-react-native/src/ios/LottieReactNative/ContainerView.swift
+++ b/node_modules/lottie-react-native/src/ios/LottieReactNative/ContainerView.swift
@@ -17,8 +17,8 @@ class ContainerView: RCTView {
         super.traitCollectionDidChange(previousTraitCollection)
         if #available(iOS 13.0, tvOS 13.0, *) {
             if (self.traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection)) {
-                applyProperties()
-                print("dark mode changed")
+                applyProperties(false)
+                print("dark mode changed")
             }
         }
     }
@@ -195,10 +195,12 @@ class ContainerView: RCTView {
         applyProperties()
     }
 
-    func applyProperties() {
+    func applyProperties(_ shouldUpdateProgress: Bool = true) {
         guard let animationView = animationView else { return }
         let isPlaying = animationView.isAnimationPlaying
-        animationView.currentProgress = progress
+        if (shouldUpdateProgress) {
+            animationView.currentProgress = progress
+        }
         animationView.animationSpeed = speed
         animationView.loopMode = loop
         if (colorFilters.count > 0) {
