diff --git a/node_modules/metro/src/node-haste/DependencyGraph.js b/node_modules/metro/src/node-haste/DependencyGraph.js
index 477a1eb..7aa76f2 100644
--- a/node_modules/metro/src/node-haste/DependencyGraph.js
+++ b/node_modules/metro/src/node-haste/DependencyGraph.js
@@ -25,6 +25,8 @@ const ModuleCache = require("./ModuleCache");
 
 const ci = require("ci-info");
 
+const crypto = require("crypto");
+
 const fs = require("fs");
 
 const path = require("path");
@@ -242,10 +244,7 @@ class DependencyGraph extends EventEmitter {
     const sha1 = this._hasteFS.getSha1(resolvedPath);
 
     if (!sha1) {
-      throw new ReferenceError(`SHA-1 for file ${filename} (${resolvedPath}) is not computed.
-         Potential causes:
-           1) You have symlinks in your project - watchman does not follow symlinks.
-           2) Check \`blockList\` in your metro.config.js and make sure it isn't excluding the file path.`);
+      return crypto.createHash('sha1').update(fs.readFileSync(resolvedPath)).digest('hex');
     }
 
     return sha1;
