diff --git a/node_modules/metro/src/node-haste/DependencyGraph.js b/node_modules/metro/src/node-haste/DependencyGraph.js
index cdd1b81..ab2c67a 100644
--- a/node_modules/metro/src/node-haste/DependencyGraph.js
+++ b/node_modules/metro/src/node-haste/DependencyGraph.js
@@ -67,6 +67,8 @@ const ModuleCache = require("./ModuleCache");
 
 const ResolutionRequest = require("./DependencyGraph/ResolutionRequest");
 
+const crypto = require("crypto");
+
 const fs = require("fs");
 
 const path = require("path");
@@ -255,9 +257,7 @@ class DependencyGraph extends EventEmitter {
     const sha1 = this._hasteFS.getSha1(resolvedPath);
 
     if (!sha1) {
-      throw new ReferenceError(
-        `SHA-1 for file ${filename} (${resolvedPath}) is not computed`
-      );
+      return crypto.createHash('sha1').update(fs.readFileSync(file)).digest('hex');
     }
 
     return sha1;
