diff --git a/node_modules/msrcrypto/dist/msrcrypto.js b/node_modules/msrcrypto/dist/msrcrypto.js
index 3789989..ea87e46 100755
--- a/node_modules/msrcrypto/dist/msrcrypto.js
+++ b/node_modules/msrcrypto/dist/msrcrypto.js
@@ -35,9 +35,12 @@ var glb = typeof global !== "undefined" ? global :
           undefined;
 
 if (typeof document === 'undefined' && glb) {
-    glb.document = {
-        attachEvent: function() {}
-    };
+    glb = {
+        ...glb,
+        document: {
+            attachEvent: function() {}
+        }
+    }
 }
 
 var msrCryptoVersion = "1.5.6";
@@ -73,7 +76,7 @@ operations.exists = function (operationType, algorithmName) {
 /// new web workers (if supported).
 var scriptUrl = (function () {
 
-    if (typeof document !== "undefined") {
+    if (typeof glb.document !== "undefined") {
         // Use error.stack to find out the name of this script
         try {
             throw new Error();
@@ -6966,9 +6969,9 @@ function MsrcryptoEntropy() {
                     if (window.addEventListener) {
                         window.addEventListener("mousemove", this.MouseEventCallBack, true);
                         window.addEventListener("load", this.LoadTimeCallBack, true);
-                    } else if (document.attachEvent) {
-                        document.attachEvent("onmousemove", this.MouseEventCallBack);
-                        document.attachEvent("onload", this.LoadTimeCallBack);
+                    } else if (glb.document.attachEvent) {
+                        glb.document.attachEvent("onmousemove", this.MouseEventCallBack);
+                        glb.document.attachEvent("onload", this.LoadTimeCallBack);
                     } else {
                         throw new Error("Can't attach events for entropy collection");
                     }
