diff --git a/node_modules/nft.storage/src/platform.js b/node_modules/nft.storage/src/platform.js
deleted file mode 100644
index adb58e5..0000000
--- a/node_modules/nft.storage/src/platform.js
+++ /dev/null
@@ -1,15 +0,0 @@
-import fetch, { Request, Response, Headers } from '@web-std/fetch'
-import { FormData } from '@web-std/form-data'
-import { ReadableStream } from '@web-std/blob'
-import { File, Blob } from '@web-std/file'
-
-export {
-  fetch,
-  Request,
-  Response,
-  Headers,
-  Blob,
-  FormData,
-  File,
-  ReadableStream,
-}
diff --git a/node_modules/nft.storage/src/platform.web.js b/node_modules/nft.storage/src/platform.web.js
index 27d9785..d88cc42 100644
--- a/node_modules/nft.storage/src/platform.web.js
+++ b/node_modules/nft.storage/src/platform.web.js
@@ -1,8 +1,21 @@
-export const fetch = globalThis.fetch
-export const FormData = globalThis.FormData
-export const Headers = globalThis.Headers
-export const Request = globalThis.Request
-export const Response = globalThis.Response
-export const Blob = globalThis.Blob
-export const File = globalThis.File
-export const ReadableStream = globalThis.ReadableStream
+const { 
+  fetch, 
+  FormData, 
+  Headers, 
+  Request, 
+  Response, 
+  Blob, 
+  File, 
+  ReadableStream 
+} = ('undefined' === (typeof globalThis)) ? window : globalThis
+
+export {
+  fetch,
+  FormData, 
+  Headers, 
+  Request,
+  Response,
+  Blob,
+  File,
+  ReadableStream 
+}
