diff --git a/node_modules/nft.storage/src/platform.js b/node_modules/nft.storage/src/platform.js
index adb58e5..27d9785 100644
--- a/node_modules/nft.storage/src/platform.js
+++ b/node_modules/nft.storage/src/platform.js
@@ -1,15 +1,8 @@
-import fetch, { Request, Response, Headers } from '@web-std/fetch'
-import { FormData } from '@web-std/form-data'
-import { ReadableStream } from '@web-std/blob'
-import { File, Blob } from '@web-std/file'
-
-export {
-  fetch,
-  Request,
-  Response,
-  Headers,
-  Blob,
-  FormData,
-  File,
-  ReadableStream,
-}
+export const fetch = globalThis.fetch
+export const FormData = globalThis.FormData
+export const Headers = globalThis.Headers
+export const Request = globalThis.Request
+export const Response = globalThis.Response
+export const Blob = globalThis.Blob
+export const File = globalThis.File
+export const ReadableStream = globalThis.ReadableStream
