diff --git a/node_modules/react-native-gesture-handler/gestureHandlerRootHOC.web.js b/node_modules/react-native-gesture-handler/gestureHandlerRootHOC.web.js
new file mode 100644
index 0000000..5ce0257
--- /dev/null
+++ b/node_modules/react-native-gesture-handler/gestureHandlerRootHOC.web.js
@@ -0,0 +1,11 @@
+/**
+ * This file will be loaded by babel when building for web instead of gestureHandlerRootHOC.js.
+ * gestureHandlerRootHOC.js imports hoist-non-react-statics which makes the build to fail.
+ */
+
+export default function gestureHandlerRootHOC(Component) {
+  // Empty implementation that just returns component directly,
+  // GestureHandlerRootView is only required to be instantiated on Android.
+  // All other targets (ios, web, macos, ect...) should do nothing.
+  return Component;
+}
