diff --git a/node_modules/react-native-image-crop-picker/ios/QBImagePicker/QBImagePicker/QBAssetsViewController.m b/node_modules/react-native-image-crop-picker/ios/QBImagePicker/QBImagePicker/QBAssetsViewController.m
index 0a9fddb..2f5f528 100644
--- a/node_modules/react-native-image-crop-picker/ios/QBImagePicker/QBImagePicker/QBAssetsViewController.m
+++ b/node_modules/react-native-image-crop-picker/ios/QBImagePicker/QBImagePicker/QBAssetsViewController.m
@@ -194,9 +194,9 @@ - (void)setUpToolbarItems
 
     // Info label
     UIColor *labelColor = [UIColor blackColor];
-    if (@available(iOS 13.0, *)) {
-        labelColor = [UIColor labelColor];
-    }
+    // if (@available(iOS 13.0, *)) {
+    //     labelColor = [UIColor labelColor];
+    // }
     NSDictionary *attributes = @{ NSForegroundColorAttributeName: labelColor };
     UIBarButtonItem *infoButtonItem = [[UIBarButtonItem alloc] initWithTitle:@"" style:UIBarButtonItemStylePlain target:nil action:NULL];
     infoButtonItem.enabled = NO;
