diff --git a/node_modules/react-native-udp/react-native-udp.podspec b/node_modules/react-native-udp/react-native-udp.podspec
index 7450cc7..6f33ab6 100644
--- a/node_modules/react-native-udp/react-native-udp.podspec
+++ b/node_modules/react-native-udp/react-native-udp.podspec
@@ -13,5 +13,5 @@ Pod::Spec.new do |s|
   s.source         = { :git => package_json["repository"]["url"], :tag => "v#{s.version}" }
   s.source_files   = 'ios/**/*.{h,m}'
   s.dependency 'React-Core'
-  s.dependency 'CocoaAsyncSocket'
+
 end
