diff --git a/node_modules/webpack-dev-server/client/utils/createSocketUrl.js b/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
index f214a2d..506ab14 100644
--- a/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
+++ b/node_modules/webpack-dev-server/client/utils/createSocketUrl.js
@@ -65,7 +65,8 @@ function getSocketUrl(urlParts, loc) {
   // they are not provided
 
 
-  var sockHost = query.sockHost || hostname;
+  var sockProtocol = protocol || loc.protocol;
+  var sockHost = query.sockHost || hostname || loc.hostname;
   var sockPath = query.sockPath || '/sockjs-node';
   var sockPort = query.sockPort || port;
 
@@ -74,7 +75,7 @@ function getSocketUrl(urlParts, loc) {
   }
 
   return url.format({
-    protocol: protocol,
+    protocol: sockProtocol,
     auth: auth,
     hostname: sockHost,
     port: sockPort,
